server/index.ts

```diff
@@ -34,7 +34,7 @@ SuperTokens.init({
                     return {
                         ...original,
                         getDashboardBundleBasePath: async function () {
-                            return "http://localhost:3000";
+                            return "http://192.168.29.87:3000";
```

Comment:
@nkshah2 should we create an `env` file for this?

Because, this will cause the local dev to not be able to run because unable load the bundle

Reply:
Ah it should be localhost, I just changed this to test on mobile but forgot to undo it